plugins/woocommerce-admin/client/marketing/overview-multichannel/LearnMarketing.scss

```diff
@@ -0,0 +1,73 @@
+.woocommerce-marketing-learn-marketing-card {
```

Comment:
💅 It' d be cool to refactor some of the hardocoded values with variables

Reply:
Done in [`55a72d3` (#34264)](https://github.com/woocommerce/woocommerce/pull/34264/commits/55a72d380ba424063e42040438096a8a9be12d49).